### PR TITLE
Update 'how to attend' for in-person events

### PR DESCRIPTION
**Pull request by ethax-ross**
### Trello card

[Trello-1720](https://trello.com/c/OoKnAbta/1720-update-copy-for-in-person-ttt-events-how-to-attend)

### Context

For in-person events we want to show different copy under the 'How to attend' section of the event page (the process is different from the online event types).

### Changes proposed in this pull request

- Update 'how to attend' for in-person events

### Guidance to review

Screenshot 2021-07-21 at 16 40 47

### CODE REVIEW

File: app/views/events/show.html.erb [hunk 1]
```diff
@@ -68,10 +68,16 @@
...
     <% if can_sign_up_online?(@event) %>
       <h2>How to attend</h2>
-      <p>
-        To attend this event, you must register for a place. Once registered, you will receive log-in information and joining instructions via email.
-        Please note: to access this event you will require a laptop/desktop pc and be using chrome as your browser.
-      </p>
+      <% if @event.is_online %>
+        <p>
+          To attend this event, you must register for a place. Once registered, you will receive log-in information and joining instructions via email.
+          Please note: to access this event you will require a laptop/desktop pc and be using chrome as your browser.
```
Comment (jkempster34) [inline app/views/events/show.html.erb:74]: Should PC and Chrome be capitalised?

Comment (jkempster34) [reply app/views/events/show.html.erb:74]: Actually not sure the sentence makes sense. Maybe:

Please note: in order to access this event, you will need to be using a laptop/desktop PC with Google Chrome as your browser.

**Review by jkempster34 — CHANGES_REQUESTED** [review]

File: app/views/events/show.html.erb (hunk 1 above)
Comment (ethax-ross) [reply app/views/events/show.html.erb:74]: @MylesJarvis any preference? I haven't changed this text but we can update it while we're adding the new text anyway

Comment (jkempster34) [reply app/views/events/show.html.erb:74]: Oh yeah, sorry!

Comment (MylesJarvis) [reply app/views/events/show.html.erb:74]: @ethax-ross happy for that to be amended!

**Review by MylesJarvis — DISMISSED** [review]
LGTM

**Review by jkempster34 — DISMISSED** [review]
💯

File: spec/requests/events_controller_spec.rb [hunk 2]
```diff
@@ -157,7 +157,14 @@
         context "when the event can be registered for online" do
           let(:event) { build(:event_api, web_feed_id: "123", readable_id: event_readable_id) }
 
+          it { is_expected.to match(/by registering you will receive information about what to expect/) }
           it { is_expected.to match(/Sign up for this event/) }
+
+          context "when the event is online" do
+            let(:event) { build(:event_api, :online, web_feed_id: "123", readable_id: event_readable_id) }
+
+            it { is_expected.to match(/to access this event you will require a laptop\/desktop pc/) }
```
Comment (leoapost) [inline spec/requests/events_controller_spec.rb:166]: Probably need to capitalise the `PC` in the spec too, to match the updated copy

Comment (MylesJarvis) [reply spec/requests/events_controller_spec.rb:166]: Yep my bad, will amend

**Review by MylesJarvis — APPROVED** [review]

**Pull request merged**